Source/SwiftLintFramework/Rules/StatementPositionRule.swift

```diff
@@ -28,15 +28,59 @@ public struct StatementPositionRule: Rule {
             "}↓  else {",
             "}↓\ncatch {",
             "}\n\t↓  catch {"
+        ],
+        corrections: [
+          "}\n else {\n":"} else {\n",
+          "}\n   else if {\n":"} else if {\n",
+          "}\n catch {\n":"} catch {\n"
         ]
     )
 
     public func validateFile(file: File) -> [StyleViolation] {
-        let pattern = "((?:\\}|[\\s] |[\\n\\t\\r])\\b(?:else|catch))\\b"
+      let pattern = "(?:\\}|[\\s] |[\\n\\t\\r])\\b(?:else|catch)\\b"
+
+      return violationRangesInFile(file, withPattern: pattern).flatMap { range in
+        return StyleViolation(ruleDescription: self.dynamicType.description,
+          location: Location(file: file, characterOffset: range.location))
+      }
+    }
+
+    public func correctFile(file: File) -> [Correction] {
```

Comment:
Actually, one more thing: just move the body of `correctFile(_:withPattern:)` to this function.